Replace usage of meshExpansion flag with istiod gateway code sample (#7869)
istio/istio.io: content/en/docs/setup/install/multicluster/shared/index.md

$ istioctl install -f istio-main-cluster.yaml --context=${MAIN_CLUSTER_CTX}
{{< /text >}}

> If using the `istio-ingressgateway` option, expose Istiod, for example by using the provided sample configuration.

Reviewer: this sentence seems like it's missing something. Maybe `you can expose istiod...`?

Also, maybe we should call it `the control plane` instead of `istiod`?

> If you're using the `istio-ingressgateway` option, expose the control plane by using the provided sample configuration.

Reviewer 2: What to do if you are NOT using the `istio-ingressgateway` option. I don't think that's possible. How about?
```suggestion
Expose the control plane by configuring the `istio-ingressgateway` using the provided sample configuration.
```

Author: In the other case (Internal Load Balancer), the control plane is already exposed by the `components` part of the install config

Reviewer 2: I see. Maybe this would be a little more clear?
```suggestion
If you selected the `istio-ingressgateway` option, expose the control plane using the provided sample configuration.
```
